src/core/transaction.zig

```diff
@@ -576,61 +583,54 @@ test "create transfer transaction" {
 }
 
 test "tmp" {
-    const msg = Message.default();
-    try std.testing.expect(msg.account_keys.len == 0);
+    try std.testing.expect(Message.EMPTY.account_keys.len == 0);
 }
 
 test "blank Message fails to sanitize" {
-    try std.testing.expect(error.MissingWritableFeePayer == Message.default().sanitize());
+    try std.testing.expectError(error.MissingWritableFeePayer, Message.EMPTY.sanitize());
 }
 
 test "minimal valid Message sanitizes" {
-    var pubkeys = [_]Pubkey{Pubkey.default()};
-    const message = Message{
-        .header = MessageHeader{
+    try std.testing.expectEqual({}, Message.sanitize(&.{
+        .header = .{
             .num_required_signatures = 1,
             .num_readonly_signed_accounts = 0,
             .num_readonly_unsigned_accounts = 0,
         },
-        .account_keys = &pubkeys,
-        .recent_blockhash = Hash.generateSha256Hash(&[_]u8{0}),
-        .instructions = &[_]CompiledInstruction{},
-    };
-    try message.sanitize();
+        .account_keys = &.{Pubkey.ZEROES},
+        .recent_blockhash = Hash.generateSha256Hash(&.{0}),
+        .instructions = &.{},
+    }));
 }
 
 test "Message sanitize fails if missing signers" {
-    var pubkeys = [_]Pubkey{Pubkey.default()};
-    const message = Message{
+    try std.testing.expectError(error.NotEnoughAccounts, Message.sanitize(&.{
         .header = MessageHeader{
```

Comment:
Can be`.header = .{`

Reply:
6f4006bdfdb1e98dc4aeee27e0405b2a69c28f6a